packages/datadog-plugin-gcloud-pubsub/test/index.spec.js

```diff
@@ -0,0 +1,153 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const { expectSomeSpan, withDefaults } = require('../../dd-trace/test/plugins/helpers')
+const plugin = require('../src')
+
+const BASE_PROJECT_ID = `test-project-${bigRandom()}-`
+let projectCounter = 0
```

Comment:
Do they have to be sequential?

Reply:
Having them sequential ensures that we won't ever run into a collision during the tests. It avoids having to call `bigRandom()` every time.

Reply:
Calling the function every time is light enough that it's fine if it keeps the tests simpler.